package.json

```diff
@@ -66,76 +66,78 @@
 		"package-lock-sanitizer": "package-lock-sanitizer"
 	},
 	"dependencies": {
-		"@sebgroup/bootstrap": "^4.0.0",
-		"chart.js": "^2.8.0",
+		"@sebgroup/bootstrap": "^5.1.0",
```

Comment:
Do you need this? this is an upgrade to bootstrap which means we need to test that all the other component still working properly

Reply:
I suggest you wait for #115  to be merged into develop and just pull the latest from develop into this branch